catalog/resource_lakehouse_monitor.go

```diff
@@ -0,0 +1,87 @@
+package catalog
+
+import (
+	"context"
+
+	"github.com/databricks/databricks-sdk-go/service/catalog"
+	"github.com/databricks/terraform-provider-databricks/common"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+type Monitor struct {
+	catalog.CreateMonitor
+	FullName string `json:"table_name"`
```

Comment:
+1 for matching the path parameter exactly. `table_name` is also kind of confusing because it might suggest that you only need the table name itself for this request, but I think you want the fully qualified name including catalog/schema, right?

Hm... @alexott we should probably handle this automatically in `StructToSchema`, right? Do you know why we ignore fields that are missing the `json` tag or have `json:"-"`?

Reply:
I think its because this field is part of the query  parameters in the URL rather than the request body

Reply:
Yes, but this structure is not used for sending requests, only for computing the TF schema.